ergoCubSN000/wrappers/motorControl/torso-mc_remapper.xml

```diff
@@ -11,7 +11,8 @@
     <param name="joints"> 3 </param>
     <action phase="startup" level="5" type="attach">
         <paramlist name="networks">
-            <elem name="torso_joints">  torso-eb5-j0_2-mc </elem>
+            <!-- <elem name="torso_joints">  torso-eb5-j0_2-mc </elem> -->
+            <elem name="torso_joints">  torso-eb5-j0_2-jtcvc  </elem>
```

Comment:
Actually I realized that even the disabled tags will not work due to these modifications in the remapper files

Reply:
So, I can foresee two paths:
1. I disable the CI for ergoCubSN000 outright.
2. We refactor the configuration a bit so that we can create the `*-mc_remapper-jtcvc.xml` files to be included within the main `ergocub.xml` file. This way, we could disable the jtcvc stuff easily.